src/fs-access/file-open.mjs

```diff
@@ -29,19 +29,19 @@ export default async (options = [{}]) => {
   if (!Array.isArray(options)) {
     options = [options];
   }
-  const types = [];
-  options.forEach((option, i) => {
-    types[i] = {
+  const types = options.map((option) => {
+    const type = {
       description: option.description || 'Files',
```

Comment:
I'd prefer to use destructuring assignment for options and move `const type` to the end of this arrow function